From: =?utf-8?q?S=C3=A9bastien_Villemot?= <sebastien@debian.org>
Date: Tue, 12 Dec 2017 00:00:00 +0000
Subject: Fix tests that check homedir attributes
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

When building Debian packages, the variable HOME may be set to a nonexistent
directory (see Policy §9.2.3).

In order to avoid test failures at build time, this patch replaces HOME by
/tmp in two tests.
---
 tests/filesys.pure.lisp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/filesys.pure.lisp b/tests/filesys.pure.lisp
index 03c8b67..96c6749 100644
--- a/tests/filesys.pure.lisp
+++ b/tests/filesys.pure.lisp
@@ -254,11 +254,11 @@
 
 (with-test (:name (file-author stringp))
   #-win32
-  (assert (stringp (file-author (user-homedir-pathname))))
+  (assert (stringp (file-author #p"/tmp")))
   #+win32
   (assert (not (file-author (user-homedir-pathname)))))
 (with-test (:name (file-write-date integerp))
-  (assert (integerp (file-write-date (user-homedir-pathname)))))
+  (assert (integerp (file-write-date #p"/tmp"))))
 
 ;;; Generated with
 ;;; (loop for exist in '(nil t)
